rfcs/text/0000-authentication-fields.md

```diff
@@ -0,0 +1,147 @@
+# 0000: Authentication Fields RFC
+
+Stage: 0
+Date: 2021-08-08
+
+This RFC will provide normalization for fields related to authentication fields to assure that they are retained primarily in ECS core and in any extended fieldset when needed. These fields are important to normalize into ECS fields in order to maximize effectiveness of cross log utilization. 
+
+## Fields
+
+#### Log
+|Proposed Field Name|Type|Value|Description|
+| --- | --- | --- | --- |
+|authentication.scope|keyword|user_impersonation|azure.activitylogs.identity.claims.http://schemas_microsoft_com/identity/claims/scope|
+|authentication.requirement|keyword|multiFactorAuthentication|azure.activitylogs.properties.authenticationRequirement|
+|authentication.additional.details|keyword|MFA requirement satisfied by claim in the token|azure.activitylogs.properties.status.additionalDetails|
+|authentication.token|keyword|AzureAD|azure.activitylogs.properties.tokenIssuerType|
+|authentication.token.identifier|keyword|MjlkY2M0MzItNWU4YS00NjU5LTlmMDMtNmVkZTE4NDAwMzAw|azure.activitylogs.properties.uniqueTokenIdentifier|
+|authentication.authorization|keyword|ROOTMANAGESHAREDACCESSKEY|azure.resource.authorization_rule|
+|authentication.isInteractive|keyword|FALSE|azure.signinlogs.properties.is_interactive|
+|authentication.risk|keyword|low|azure.signinlogs.properties.risk_level_aggregated|
+|authentication.id|Unique ID given across primary authentication and additional (multi factor) authentication.|||
+|authentication.protocol|Authentication Protocol (authproto)|||
+|authentication.scope|Requested scope of user|keyword| Defines scope of authentication request|
+|authentication.requirement|Login Authentication Requirements|keyword| What is required to authenticate||
+|authentication.additional.details|Login Additional Details|list| Important login details not applicable in other areas|
+|authentication.token|Type of token for the login event|keyword| Unique token provided during the event|
+|authentication.token.identifier|Login token identifier/value|keyword| Unique identifier|
+|authentication.authorization|Login Authorization Information|keyword| Important authorization information|
+|authentication.isInteractive|Is the login an interactive login?|boolean| Describes whether the request was an interactive authentication or not|
+|authentication.risk|Risk level for the authentication event.|keyword| This describes the risk level for the authentication event.|
+|authentication.id|Unique ID given across primary authentication and additional (multi factor) authentication.| keyword | This unique value allows for identification of the authentication request|
+|authentication.protocol|Authentication Protocol (authproto)|keyword|This field provides information about the authentication protocol used.| 
```

Comment:
It appears these are all duplicate field entries, but with slightly misaligned information (description in the keyword column).